### ISSUE TITLE
How many frames can be processed per second

### ISSUE THREAD

**Issue by MingRongXi** [saswat0/License-Plate-Recognition#8]
Sorry, saswat0, I clicked in a wrong place that closed the issue. My cuda version is 10.2, so I install the cudatoolkit and cudnn with the command: conda install cudatoolkit=10.2, conda install cudnn. Because the first command specifies the cudatoolkit version, so the cudnn 7.6.5 is installed. Then I install the tensorflow-gpu with conda install tensorflow-gpu. The tensorflow-gpu 1.8 is installed. But when I run the python video.py, the gpu is not used.

**Comment by saswat0** [comment]
Try installing tensorflow-gpu using pip instead of conda. That'll download and initialise cuda+cudnn separately
Also, ensure that GPU and CUDNN flags are set to 1. Run `make clean` in place of `make` to clear out any persisting build components from the previous make

**Comment by MingRongXi** [comment]
I reinstalled tensorflow-gpu, keras and cudnn. The speed is faster than before, but the fps is still low. It spends 1 or 2 seconds processing an image. I run nvidia-smi to check the gpu usage, memory-usage: 6365MiB /  8192MiB,   GPU-Util: 80%.

**Comment by saswat0** [comment]
If the GPU usage is ~80%, I think that's the maximum fps that you could get from your current system. Do mind that this is an academic paper implementation and hence focuses more on accuracy and applicability rather than deployability/performance
Since further gain in speed won't be possible considering the current hardware bottlenecks, what I could possibly suggest you to do is to check out the 'tensorrt' branch of this repository. That implementation will hopefully provide you with better fps with the same hardware
